sdk/servicebus/service-bus/tsconfig.json

```diff
@@ -36,7 +36,9 @@
     "node_modules",
     "typings/**",
     "./samples/**/*.ts",
-    "./test/perf/azure-sb-package/*.ts"
+    "./test/perf/azure-sb-package/*.ts",
+    "test/perf/*",
+    "test/stress/*"
```

Comment:
Just calling out..
Have not updated the tests under `test/stress` and `test/perf` to track 2 and excluded from the tsconfig.
If that's ok, I'll log a separate issue and we can revisit to this once the top-level client version is implemented.

Reply:
https://github.com/Azure/azure-sdk-for-js/issues/7727